src/Common/Constants.ts

```diff
@@ -96,7 +96,6 @@ export class Flights {
   public static readonly AutoscaleTest = "autoscaletest";
   public static readonly PartitionKeyTest = "partitionkeytest";
   public static readonly PKPartitionKeyTest = "pkpartitionkeytest";
-  public static readonly Phoenix = "phoenix";
```

Comment:
>Phoenix [](http://example.com/codeflow?start=25&length=7)

why do you want to remove this? let me sync with you offline

Reply:
ok

Reply:
Please dont remove the flight. We will be flighting when we roll out in prod.